src/Chapter1.hs

```diff
@@ -473,7 +473,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = mod n 10
```

Comment:
Almost there! Unfortunately, it won't work properly on negative numbers. You would need to use one more standard function 🙂